### ISSUE TITLE
Please render natural=shrubbery

### ISSUE THREAD

**Issue by hungerburg** [gravitystorm/openstreetmap-carto#4473]
Since hedges do not render as areas, cf. https://github.com/gravitystorm/openstreetmap-carto/pull/3844, a limited but highly prolific number of people use natural=scrub to map, what are in my understanding mostly hedge-like features: barriers to keep pedestrians from taking shortcuts. See https://wiki.openstreetmap.org/wiki/Tag:natural=shrubbery for complete description and account of other use cases.

As of now, this custom is regionally limited, the Netherlande being a prime spot of exploding use: https://wiki.openstreetmap.org/wiki/File:ScrubByNumbersNLvsAT.png shows a comparison with use in Austria. In Austria, like in most of the world, such "shrubberies" are not mapped at all, not the least maybe, because there exists (not until recently) no valid tagging for them.

As the trend to map micro features is expected to continue ever stronger, there will be demand for a tag to correctly map those "shrubberies". As the people doing such mappings are very sensitive to rendering, it is to be expected that they will not use the concise tag, but the one that renders.

Therefore my plea to render natural=shrubbery. Give the authors of the shrubbery proposals a chance of having created a successful tag. Apart from preventing the information loss by dilution of millions of currently concise mappings of actual scrub, I think also the cartographers work will be a little easier, if a tag, that starts rendering at z7 or so will not be used on perhaps even more millions of features, that range from closet sized to the size of a medium flat.

**Comment by imagico** [comment]
Very new tag, zero uses before February this year, no consensus on its use.

Current use (around 2000 times) is fairly consistently applied for urban greenery.  But to a significant extent for polygon mapping of linear hedges for which linear barrier=hedge (+width=*/height=*) is the more established (and semantically more meaningful) method of mapping, see for example:

https://www.openstreetmap.org/way/914194296
https://www.openstreetmap.org/way/562827568
https://www.openstreetmap.org/way/926141288
https://www.openstreetmap.org/way/916799341
https://www.openstreetmap.org/way/917567385

I am not opposed to rendering this in principle but right now this tagging competes with barrier=hedge (on linear ways), natural=scrub and natural=heath and to some extent also leisure=garden.  I think it needs to establish its meaning and use relative to these tags with some clarity and acceptance among mappers before we can consider supporting it here.

**Comment by jeisenbe** [comment]
This tag is currently only used in the Netherlands, England and a couple other spots. It needs to be much more widely used before it could be rendered here: 
<img width="729" alt="Screen Shot 2021-09-18 at 23 03 49" src="https://user-images.githubusercontent.com/42757252/133917247-17d51024-251a-4dc2-93bb-1eef130f1e67.png”>

I believe we should close this issue for now, but it could be reopened in a year or two if the situation has changed and the tag has been widel adopted by mappers in many countries.

**Comment by endim8** [comment]
I think this should be reopened, natural=shrubbery is now consistently used for an area of shrubs, and the meaning is clearly stated on https://wiki.openstreetmap.org/wiki/Tag:natural%3Dshrubbery

it is also being adopted very quickly, as can be seen from the chronology graph:
<img width="344" alt="Screenshot 2022-04-04 at 13 26 39" src="https://user-images.githubusercontent.com/26939824/161543379-8eb24136-c79f-422f-9815-e64e25c798ee.png">

Adding Carto rendering would be beneficial as it would prevent tagging for the renderer in lots of cases, for example i've seen a lot of areas that should be tagged shrubbery tagged as grass, scrub, garden etc.

**Comment by kocio-pl** [comment]
OK, it's over 5k now. This should be quite easy to implement, is anyone ready to prepare PR?

**Comment by imagico** [comment]
Note the assessments from me in https://github.com/gravitystorm/openstreetmap-carto/issues/4473#issuecomment-921750738 and from @jeisenbe in https://github.com/gravitystorm/openstreetmap-carto/issues/4473#issuecomment-922421023 still apply.  The use of the tag has slightly increased but it still has no broader acceptance.  It is used by individual mappers in localized concentrations but there are no places where it is the dominating tagging for anything specific.

Overall at the moment where it is used it is essentially used as an umbrella tag for urban and less frequently near urban rural greenery.  Its use in particular overlaps with what is much more commonly tagged (roughly in that order of prevalence):

* `leisure=garden`
* `natural=heath`
* `natural=scrub`
* `barrier=hedge` - either instead of what is more commonly mapped with linear hedges (like https://www.openstreetmap.org/way/1015021223) or in addition to linear hedge mapping (like https://www.openstreetmap.org/way/1028424094)
* `natural=shrub` (like here: https://www.openstreetmap.org/way/1020960779)
* `natural=forest` (including double tagging like here: https://www.openstreetmap.org/way/918418611)

and just as a generic urban greenery tagging (like https://www.openstreetmap.org/way/1046678798)

In other words:  Its use has semantically widened compared to my previous assessment from last September but is is not in any way better defined in its delineation towards those tags which are much more commonly used for urban greenery and in a much better defined way.  Like i wrote above:

> I think it needs to establish its meaning and use relative to these tags with some clarity and acceptance among mappers before we can consider supporting it here.

And to be clear (because this seems to sometimes cause misunderstanding): This refers to the de facto meaning and de facto acceptance among mappers.

**Comment by endim8** [comment]
> Overall at the moment where it is used it is essentially used as an umbrella tag for urban and less frequently near urban rural greenery.

That's what a shrubbery is, yes.

  Its use in particular overlaps with what is much more commonly tagged (roughly in that order of prevalence):
> 
> * `leisure=garden`
> * `natural=heath`
> * `natural=scrub`
> * `barrier=hedge` - either instead of what is more commonly mapped with linear hedges (like https://www.openstreetmap.org/way/1015021223) or in addition to linear hedge mapping (like https://www.openstreetmap.org/way/1028424094)
> * `natural=shrub` (like here: https://www.openstreetmap.org/way/1020960779)
> * `natural=forest` (including double tagging like here: https://www.openstreetmap.org/way/918418611)

These are all used interchangeably because there is no tag for shrubbery. when you look at the areas they describe on aerial imagery, they are all the same sort of thing. (apart from natural=forest, that should be grass)

> In other words:  Its use has semantically widened compared to my previous assessment from last September but is is not in any way better defined in its delineation towards those tags which are much more commonly used for urban greenery and in a much better defined way.  

I disagree entirely, the use is semantically widening because people want to use a tag that renders, so when natural=shrubbery doesn't, they use something similar. If natural=shrubbery was only used to mean the same thing as landuse=grass, for example, the tag wouldn't be proposed.

**Comment by jdhoek** [comment]
> The use of the tag has slightly increased but it still has no broader acceptance. It is used by individual mappers in localized concentrations but there are no places where it is the dominating tagging for anything specific.

Well yeah, without rendering in what is the face of OpenStreetMap, using this tag is pioneering. I am pleasantly surprised it growing as fast as it given the lack of rendering.

Most mappers just don't bother and use `natural=scrub` for these planted and managed shrubs, and for hedges mapped as areas too (especially with the support for rendering `barrier=hedge` withdrawn in Carto). It sucks having to use tags that don't render when your local newspaper even uses OpenStreetMap with the Carto layer for its articles, and I can understand the pragmatic approach of these mapper. But is this what we want to stimulate mappers to do?

There are a bunch of mappers here who — while not necessarily agreeing with the method of just removing rendering — at least understand your viewpoint regarding hedges mapped as areas and the misapplication of `area=yes`, and who find that misusing `natural=scrub` for urban managed greenery is hardly ideal from a semantic perspective. `natural=shrubbery` is at least an effort to solve these issues.

**Comment by imagico** [comment]
For better understanding (we should really put this into some kind of FAQ document because it turns up so frequently):  One of [our core goals](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CARTOGRAPHY.md) is to support mappers in consistent use of tags and prevent unfavorable fragmentation of tagging.  So if we'd communicate to mappers:  You can either map urban greenery in a semantically well defined and differentiated fashion using leisure=garden, natural=heath, natural=scrub, barrier=hedge, natural=shrub etc. like the vast majority of other mappers do or you can tag it undifferentiatedly as  natural=shrubbery then we'd have failed fundamentally in doing our job. 

As i have expressed above i am in principle not opposed to rendering `natural=shrubbery` if it develops into a tagging for specific forms of urban greenery (in particular for what `barrier=hedge` has been partly overloaded in use on polygons in the past leading to semantic ambiguities) and consensus develops among mappers about the semantic delineation of this towards other tags.  If any trend would become visible in direction of a well defined meaning we could try to think of a rendering that supports specifically such use and discourages a broader generic use for any and all urban greenery.  But so far there unfortunately is no indication that something like this is happening.  Instead the most dominant use of the tag is generic decorative greenery, mostly not very tall, i.e. heath height, which otherwise is predominantly tagged `leisure=garden` or `natural=heath`.

**Comment by hungerburg** [comment]
Do I follow up correctly: natural=heath suggested tagging for groundcover berberis eg?

**Comment by imagico** [comment]
> Do I follow up correctly: natural=heath suggested tagging for groundcover berberis eg?

Well - what i suggest is not really the point here - `natural=heath` is however quite frequently used in urban contexts - often for maintained features.  Like for example:

https://www.openstreetmap.org/way/321018325
https://www.openstreetmap.org/way/572438946
https://www.openstreetmap.org/way/1039907307
https://www.openstreetmap.org/way/995551598
https://www.openstreetmap.org/way/633616777
https://www.openstreetmap.org/way/925076690

**Comment by imagico** [comment]
See also #4251 by the way - where we have considered and positively assessed rendering `landuse=flowerbed` as a well defined and consistently used tag for a specific type of urban greenery.  If use of `natural=shrubbery` would develop in a similarly positive fashion both could be rendered - provided that a suitable design can be found that fits into our design schema and that is intuitively understandable.

**Comment by hungerburg** [comment]
I clicked all the links. I guess, that all the objections against rendering this tag here would just the same apply on the usage of the heath tag there, if it was not already in wide use for something different.

**Comment by imagico** [comment]
`natural=heath` as a tag has its own issues, in particular its quite frequent use for mapping herbaceous vegetation (see [here](https://www.openstreetmap.org/user/SK53/diary/397927) for some more lengthy discussion).  But it is overall still fairly consistently used for something rather specific - that is low growing woody vegetation.  I can't say for sure without on-the-ground inspection if all the examples match this core use but it is quite possible that they do.

But lets not get side tracked - the fact that we certainly have tags that we render for historic reasons despite them being no more used in a well defined fashion (which as indicated does not apply for `natural=heath`) is not a good reason to start rendering further tags with similar problems.

**Comment by jdhoek** [comment]
How is `natural=sbrubbery` not well defined and consistently used? All tags have examples of not being used correctly, even `landuse=flowerbed`. If you use that as a criterium no new tag can ever be included in Carto.

**Comment by imagico** [comment]
The question if a certain tag is used correctly by some subjective opinion about the meaning of a tag is off-topic here.  And as explained above we have clear consensus that `landuse=flowerbed` is used quite consistently with a fairly precise meaning.  

`natural=shrubbery` OTOH is - as explained - used in a broad meaning for any and all kinds of urban and near urban greenery (anything that is alive and green in an urban or near urban environment) - but only in a very sporadic and patchy fashion by individual mappers while most other mappers use more specific tags (see above) to map the same things.  Within this semantic domain of _generic urban greenery_ so far no clear core focus of the tag on something more specific that we could realistically support by rendering it, like for example the polygon mapping of `barrier=hedge` like stuff (i.e. fairly compact and homogeneous scrub plantations) can currently be observed.  As much as i regret this on some level (because i think it would be highly beneficial if mappers had well established and nuanced ways to map plants in urban environments - be that in the form of new primary tags like this or secondary tags to already existing features) this is - for the moment - a reality we cannot ignore.

Side note:  I have observed the struggles of mappers to invent new tags within the OSM tagging scheme in recent years on many occasions and in particular what typically does not work these days any more although it might have worked in the early years of OSM.  I have written down [some advice based on these observations last year](http://blog.imagico.de/how-to-invent-new-tags-in-2021/) to help mappers inventing new tags to avoid the most likely errors.

**Comment by vincentvd1** [comment]
> Overall at the moment where it is used it is essentially used as an umbrella tag for urban and less frequently near urban rural greenery. Its use in particular overlaps with what is much more commonly tagged (roughly in that order of prevalence):
> 
> * `leisure=garden`
> * `natural=heath`
> * `natural=scrub`
> * `barrier=hedge` - either instead of what is more commonly mapped with linear hedges (like https://www.openstreetmap.org/way/1015021223) or in addition to linear hedge mapping (like https://www.openstreetmap.org/way/1028424094)
> * `natural=shrub` (like here: https://www.openstreetmap.org/way/1020960779)
> * `natural=forest` (including double tagging like here: https://www.openstreetmap.org/way/918418611)

It is not really fair to address the double tagging here. While the rule is: no tagging for the renderer; a lot of mappers still do, that is just a fact. A few months back, I saw a changeset with the text "shrubbery -> scrub". So the user found shrubbery the correct tag but because it didn't render, he retagged it. I later explained the situation and the tagging rule above to him and he is correctly using shrubbery now. So I know for sure that when we get render support on Carto, the (even more correcter) use will increase. I have tagged a lot of `natural=scrub` myself because there was no other tag like `natural=shrubbery`

And wiki pages are not static. They change over time with new insights. Over time, we can clearify some examples and definitions but I don't see a significant misuse of the tag yet. And a lot of people already ackowledged the gap that `shrubbery` will fill and support this tag (the very steep graph supports this despite lack of rendering).  Render support will allow the tag to grow further and to mature just like many other tags.

**Comment by hungerburg** [comment]
I fully support the reasoning of imagico. A tag gets the meaning from its usage. I do feel bad though, when this shines a light on the openstreetmap community, that they differentiate vegetation solely by how tall it grows. A heath, a scrub, a wood are much more that different height plants. They are home to different kind of animal species too. They do well in certain climates more than in others, and so on. A single tag combination "natural=vegetation+height=nn" does not do the same job as tagging heath, scrub or wood.

**Comment by tjur0** [comment]
I can see where imagico is coming from.

I agree that `natural=shrubbery` had a wider use than the more specific tags like `natural=flowerbed` or `barrier=hedge`. However, the majority of `natural=shrubbery` is not used as general urban greenery. It is used as a specific type of plant, with a specific sociological meaning.

I do think some of the usage of `natural=shrubbery` is incorrect, and I think that we should correct these mistakes. But these mistakes are in the minority. The majority of `natural=shrubbery` is correctly mapped.

It is unfair to discredit a complete tag based on a small percentage of the usage.

I completely disagree with the suggestion that the objects that currently are tagged with `natural=shrubbery` could be tagged with another existing tag. And suggesting this shows a lack of understanding towards the different vegetation in the streetscape.

According to the usage, most of the `natural=shrubbery` is used in urban environments. Specifically on the side of streets and roads. Therefore, these objects are actively maintained and artificial planted by humans, this fact alone rules out both `natural=shrub` and `natural=heath` as a substitute for these objects.

Another suggestion is to use `leisure=garden`, that suggestion is even less appropriate, a shrubbery is not a garden. Sure, a garden could contain shrubberies. In the same way a garden could contain trees, grass, and water. But a shrubbery is not meant to be visited or to be walked through.

Tagging shrubberies with any tag of the above would be a clear example of tagging for the renderer.

`natural=shrubbery` is specifically used as the maintained and artificial counterpart of natural=shrub. And is therefore not any wider than `natural=shrub` in any way.

The fact that `natural=shrubbery` is used in localized concentrations should not be a reason not to render natural=shrubbery. the localized concentrations are a result of the low number of mappers willing to use a tag that does not get rendered. And by the large number of shrubberies in the world.

I think this tag should be rendered, mainly because there is no alternative tag to map shrubberies.

**Comment by tjur0** [comment]
with respect to you

> Please no tagging discussion here

I did not start that discussion 

> provide evidence and not just make unsubstantiated claims.

That is a bit hypocritical, data is evidence, finding and linking a handful of ways is not

**Comment by hungerburg** [comment]
Actually, in the colloquial meaning, a shrubbery IS a type of garden. Just search youtube for "shrubbery monty pythons", and you can see. Too bad, that some time in the past, mappers did not use "barrier=hedgerow", because then "barrier=hedge" would be free now for polygons.

**Comment by jdhoek** [comment]
Here is another sample rendering from the PR in #4530. (It includes some patches of `natural=scrub` that should become `natural=shrubbery`, but I tend to leave existing entities as is if the replacement leads to a degraded map in OpenStreetMap's showcase rendering).

![Screenshot from 2022-04-13 18-15-34](https://user-images.githubusercontent.com/683699/163233226-2fea585a-fd05-4d42-a0b6-d4d650cd013a.png)

**Comment by endim8** [comment]
I still don't understand your argument as to why this shouldn't be rendered. Honestly, the reason Carto is stagnating (no changes for at least 5 months!) is that valid requests like this are shot down by the maintainers. 
If a tag isn't rendered, the majority of mappers just won't use it. I understand that Carto doesn't want to influence tag preferences, but this is taken to an extreme which just prevents new tagging & rendering.

**Comment by XandrexOSM** [comment]
Hello,

I have a different approach.
If I look at the vegetation values for natural=*, specifically for those which can be used for a surface, all are mapped in openstreetmap-carto except :
- the ones which must not be used (fell and moor)
- one with less than 1000 occurences in OSM (tundra)
- shruberry

So a different approach could be to map all the values for the vegetation section in the wiki page https://wiki.openstreetmap.org/wiki/Key:natural#Vegetation in a generic green, except those for which it has been decided to use an explicit different rendering.

After reading all the comments, I still do not understand what prevents the rendering of shruberry in openstreetmap-carto. I can understand that it is low priority, but refuse to render it is very different.

**Comment by patrickov** [comment]
I like to talk with examples.
Let me ask @imagico about the following.

For the green space in the middle of the following view:
https://www.google.com/maps/@22.3535023,114.1091815,3a,75y,290.27h,98.67t/data=!3m7!1e1!3m5!1seI3hiQw306PLqPPOhbPqlQ!2e0!6shttps:%2F%2Fstreetviewpixels-pa.googleapis.com%2Fv1%2Fthumbnail%3Fpanoid%3DeI3hiQw306PLqPPOhbPqlQ%26cb_client%3Dmaps_sv.tactile.gps%26w%3D203%26h%3D100%26yaw%3D71.305534%26pitch%3D0%26thumbfov%3D100!7i16384!8i8192

1.  Is natural=shrubbery a correct tag to use?
2.  If yes, do you think that it's not significant enough to be rendered on the carto map?
3.  If no, what tag should I use?


Also, what about the green island in the middle of the following roundabout?
https://www.google.com/maps/@22.3523008,114.1070948,3a,75y,4.96h,105.99t/data=!3m6!1e1!3m4!1s66Sh1K2tx7R5j4jBi6nvdQ!2e0!7i16384!8i8192

Or the one in the middle of the following picture?
https://www.google.com/maps/@22.3723633,114.1055946,3a,73.3y,254.46h,91.24t/data=!3m7!1e1!3m5!1sSHaC77PCZiaEmyCYnINTTg!2e0!6shttps:%2F%2Fstreetviewpixels-pa.googleapis.com%2Fv1%2Fthumbnail%3Fpanoid%3DSHaC77PCZiaEmyCYnINTTg%26cb_client%3Dmaps_sv.tactile.gps%26w%3D203%26h%3D100%26yaw%3D164.79945%26pitch%3D0%26thumbfov%3D100!7i16384!8i8192

Mind you, all these are within 3 km of my home so I can easily find more examples.

**Comment by hungerburg** [comment]
The Hongkong streetviews and a recent blog post by imagico did ring a bell in my mind: I herewith propose, that OSM does not need "natural=shrubbery", but instead, it needs

"landuse=brush", c.f. https://www.merriam-webster.com/dictionary/brush - and yes:

1) landuse, because these are planted vegetation, for THE USE of deterring entry to the land, either as a kind of barrier mapped as an area, AROUND the land, or simply the land itself
2) brush, because the word is in singular :)

I do not think, @patrickov wants to map the trees inside the roundabout as shrubbery, rather the ground cover on the road side…

**Comment by jeisenbe** [comment]
https://taginfo.openstreetmap.org/tags/?key=natural&value=shrubbery#map
This tag is still not widely used, it is popular in a couple of countries in Europe

<img width="739" alt="Screen Shot 2022-06-05 at 23 00 19" src="https://user-images.githubusercontent.com/42757252/172103998-8442ec92-ef2c-4c07-82ce-0cc973062535.png">

Note that we also do not render `natural=fell` or `natural=tundra` or `natural=desert` because those tags also do not have a clear definition for one type of vegetation.

In the case of `natural=shrubbery` it should be noted that there were many objections to this tag during the proposal process, which may still be valid, summarized here: https://wiki.openstreetmap.org/wiki/Proposed_features/shrubbery and also in the original voting: https://wiki.openstreetmap.org/w/index.php?title=Proposed_features/shrubbery&oldid=2130624

There were also objections to using `natural=scrub` + `maintained=` instead: https://wiki.openstreetmap.org/w/index.php?title=Proposed_features/shrubbery&oldid=2188267

In this case there does not seem to be consensus by mappers if `natural=shrubbery` or `natural=scrub` or `barrier=hedge` as an area or some other tag is correct. We may need to wait for longer to see if a clear consensus develops before promoting a new tag.

**Comment by jdhoek** [comment]
> In this case there does not seem to be consensus by mappers if natural=shrubbery or natural=scrub or barrier=hedge as an area or some other tag is correct. We may need to wait for longer to see if a clear consensus develops before promoting a new tag.

The proposal summary page summarizes the issue well enough: both votes had majority support, but not super-majority support. A significant percentage of mappers wants to use (or keep using) `natural=scrub` for the use proposed by `natural=shrubbery`, many driven by the fact that the former renders here. Those mappers are fully supported by Carto by default (the rendering was already there).

With `natural=shrubbery`, we are growing support organically, which given the results of the voting process — i.e., not outright dismissal, but a fundamental disagreement about which tagging to use ([see the four positions outlined here](https://wiki.openstreetmap.org/wiki/Proposed_features/shrubbery)) — is fair enough. At this point however, it would be fair to level the playing field by rendering `natural=shrubbery`. Carto shouldn't push new tags in the embryonic stage, but it shouldn't unreasonably resist tags gaining acceptance either.

The two major approaches (keep on using (or abusing depending on the point of view) `natural=scrub`, or use `natural=shrubbery`) don't clash in terms of rendering, and rendering does more good than harm (including offering mappers a solution for hedge areas, which were broken in Carto some years ago now).

**Comment by jdhoek** [comment]
>  This tag is still not widely used, it is popular in a couple of countries in Europe

To be fair: that distribution roughly corresponds with areas of the globe that are mapped in greater detail (due to the availability of mappers and mapping resources such as high resolution imagery and government-maintained background layers) where you would use `natural=shrubbery`. If you are mapping a remote village in Africa, changes are the shrubbery in the village square are not high on your list of priorities. `natural=scrub` is used globally, because it is used for quite large areas of natural scrub.

**Comment by hungerburg** [comment]
I think, jeisenbe did a good summary. The picture even shows the distribution more spread, than it actually is. More than half of the usage is in the Netherlands, https://taginfo.geofabrik.de/europe/netherlands/tags/natural=shrubbery and looking closer, there it is highly concentrated to small areas too.

My best wishes to the proponents of the tag, that they get the mapping community to produce a clear picture, of what on the ground feature the tag is to portray. For the time being, I do not see this issue worth kept open. When new data arrives, discussion can start afresh and not loaded with this here.

**Comment by hungerburg** [comment]
Thats a staggering growth. The number of entities might already show in what direction users drift, so documentation can get adapted and made more precise?

**Comment by endim8** [comment]
given that usage is now at 21k, maybe this should be reopened?

**Comment by imagico** [comment]
A reminder to everyone:  This issue tracker is for volunteer map designers to discuss map design work under the [goals of this style](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CARTOGRAPHY.md).  The challenges of this w.r.t. this specific request have been outlined above so please limit your commentary to things that might help addressing these issues, specifically design ideas that might do so, or pointers to changes in the de-facto meaning of the tag that might be significant.

I am going to try answering the question by @pelderson:

> How much growth do we need to get areas of shrubbery, mapped as natural=shrubbery, rendered?

Areas of shrubbery (according to how Wikipedia/OED define this term) are rendered when mapped in the currently predominant ways of mapping them in OSM, that is differentiated according to either height of vegetation or function (as explained in detail above):

* `leisure=garden` - used for shrubbery (but also for other vegetation) planted and maintained systematically for decorative purposes.
* `natural=heath` - used generically for low height shrubbery independent of function.
* `natural=scrub` - used generically for tall shrubbery independent of function.
* `barrier=hedge` - used for shrubbery planted in a linear arrangement to serve as a barrier of some sort.
* `landuse=forest` - used sometimes (but not exclusively) for tall shrubs planted for wood production.

`natural=shrubbery`, which is used as an alternative way to map shrubbery by some mappers, undifferentiated by either height or function and not exclusively for shrubbery but essentially for any kind of urban and near urban vegetation in height less than mature trees, is currently not rendered.

If the use of this tag would increase to a comparable or higher level than the other tags mentioned that we render in the domain of mapping areas of shrubbery without a change in de facto meaning (which is very unlikely to happen, typically tags change in meaning if their use widens) that would face us with a very difficult decision:

* we would either need to continue explicitly not rendering a tag with wide use because it cannot be consistently integrated into our current paradigm of rendering vegetation or
* we would need to completely abandon our current paradigm of rendering vegetation differentiated by physical characteristics and function and flatten all vegetation rendering in height less than mature trees to one uniform  design.  That would of course be a sad day for anyone interested in differentiated rendering of vegetation.

Again - this is unlikely to happen because at least historically the world wide mapper community has often shown to be able to rectify gross inconsistencies in tagging paradigms like this once tags really get exposed to and scrutinized by the world wide community in all its diversity.

I am confident that this will happen also here.  It could likely happen either by mappers re-defining and better differentiating and semantically delineating `natural=shrubbery` or by abandoning it in favor of other mapping ideas.  I will continue to observe this and if and when some trend crystalizes here that we can positively support under our goals i would be more than happy to support anyone in developing a rendering solution for that in OSM-Carto.

**Comment by pelderson** [comment]
> * `leisure=garden` - used for shrubbery (but also for other vegetation) planted and maintained systematically for decorative purposes.
> * `natural=heath` - used generically for low height shrubbery independent of function.
> * `natural=scrub` - used generically for tall shrubbery independent of function.
> * `barrier=hedge` - used for shrubbery planted in a linear arrangement to serve as a barrier of some sort.
> * `landuse=forest` - used sometimes (but not exclusively) for tall shrubs planted for wood production.
> 
> `natural=shrubbery`, which is used as an alternative way to map shrubbery by some mappers, undifferentiated by either height or function and not exclusively for shrubbery but essentially for any kind of urban and near urban vegetation in height less than mature trees, is currently not rendered.

leisure=garden and landuse=forest are not ways to tag areas of shrubbery. 
barrier=hedge is a linear feature, not a way to tag areas of shrubbery.
So we are left with heath vs scrub.
natural=heath is not for all low height shrubbery, it specifically tags areas of dwarf shrubs generally known as heath. 
natural=scrub is defined for osm as ".. to tag areas of uncultivated land covered with shrubs, bushes or stunted trees." 

There is a gap. There are many areas of shrubbery not covered by these two: not dwarf shrubs and not uncultivated land. They are found in large quantities in urbanised areas, as are other types of greenery. Heath is practically never used for this. Some mappers think scrub should be extended to cultivated land and groomed shrubbery, but this is not generally adopted in practice, I think because it doesn't look green enough on the map. 
This is the gap filled by the tag natural=shrubbery. It's 'urban greenery' of the shrubbery type. Other urban greenery types are catered for by several grass tags and tree tags, even flowerbeds have a tag, but the ubiquitous shrubbery type is missing.

Massive tagging for the renderer has occurred, tagging these pieces of shrubbery as gardens, village greens, hedges, parks to make them appear green on the map, as mappers feel they should. At the same time the tag natural=shrubbery has been applied with a stunning and continuing growth in usage, despite not being rendered. 
Cultivated shrubbery is tagged almost exclusively to improve the map, to make it better represent the real world. 
Once natural=shrubbery is rendered, mappers can stop mapping for the renderer, for these patches of shrubbery.
Mappers will of course differ about some areas, whether it's wild shrubbery or tamed scrub, but in general the difference is clear enough. Heath will not be confused with shrubbery.
About the rendering: filling the area with the same green colour as hedge would be fitting and fine for a general purpose map, disregarding any density tags.

**Comment by jdhoek** [comment]
> barrier=hedge is a linear feature, not a way to tag areas of shrubbery.

This of course is the standpoint of Carto as well. `barrier=hedge` combined with `area=yes` was deliberately removed from rendering, and solutions for tagging hedges mapped as areas requested. `natural=shrubbery` is one such solution, because a hedge is of course nothing more than cultivated shrubs, often planted densely (hence `shrubbery:density=dense`).

> About the rendering: filling the area with the same green colour as hedge would be fitting and fine for a general purpose map, disregarding any density tags.

Pragmatic with no downsides.

**Comment by endim8** [comment]
> > * `leisure=garden` - used for shrubbery (but also for other vegetation) planted and maintained systematically for decorative purposes...

spot-on analysis, thank you 👍

**Comment by OttoROSM** [comment]
> > leisure=garden and landuse=forest are not ways to tag areas of shrubbery...

A Garden, wether residential or municipial and regardless of private or public access, typically is of a higher quality. Although when used residential; it might also mean the residents could have paved it over with tyles and just put a pot with a plant somewhere. When used municipial, the plants and design is of a higher quality. A grouping of some shrubs, with one or two trees, would still be maintained and kept orderly, but is not a garden; it is a shrubbery. And scrubs you can find on barrenland, it will grow organically and will not be kept tidy. Sometimes part of a forestry aream where there are open areas without trees.

A Forest of course is being missused often as wel to show that an area has multiple trees standing on it; but imho a patch of grass in an urban environment with 4 trees grouped together, does not qualify as a forest. It should be landuse=grass and I would individually put the trees in it. But for the sake of mapping simplicity, a small patch of land with multiple trees and a lot of undergrowth would be tagged as "forest"; as that is what comes closest to what is there. (or shrubbery with trees?)

Similarly a patch of grass with some trees, imho does not qualify as a park. A park is often a location to travel to where multiple patched of grass, shrubbery (scrub?), trees and often water features can be found.

**Comment by imagico** [comment]
Please keep the discussion on topic, what tags are supposed to be used in what way is not a suitable subject here.  Further analysis of how the different tags discussed are de facto used in different parts of the world and to what extent that use is changing is welcome.

**Comment by OttoROSM** [comment]
> Please keep the discussion on topic, what tags are supposed to be used in what way is not a suitable subject here. Further analysis of how the different tags discussed are de facto used in different parts of the world and to what extent that use is changing is welcome.

Well that's not entirely fair; since you yourself mentioned "this tagging competes with barrier=hedge (on linear ways), natural=scrub and natural=heath and to some extent also leisure=garden. I think it needs to establish its meaning and use relative to these tags with some clarity and acceptance among mappers before we can consider supporting it here."

So I agree that the discussion around which tag should be used for what purpose; should be done on the OSM Fora or OSM Wiki and not necesarily here; but from the above and further discussion onwards, it was pulled into the matter wheter it should be rendered or not.

And IMHO to what extent things are being rendered _is_ quire important when you create maps. Especially since carto is the default renderer for OSM. And it also drives people to use tags accordingly. As shrubbery does not show up, people tend to use village green, garden, scrub, hedge+area and so forth quite inconsitently; where shrubbery would solve this.

Carto's choice is now to not render this tag at al; making for some quite odd maps when used and a mixed bag of other tags when people "map with the renderer in mind". In so far you want to create a seperate graphic representation for Shurbbery and what that would look like, I'll leave up to designers; but to not have it rendered at all seems odd.

**Comment by hungerburg** [comment]
In the area of my local knowledge, we have lots of scrub, in the sense of uncultivated dwarf woods. Curiously, the tag is not used to map hedges or manicured urban green. Perhaps, nobody realized yet, that this is actually the same, because it renders the same in OSM-Carto, i.e. some shade of green? Perhaps people consider this something different and rather map nothing unless there is a way to map correctly, i.e. with a different pattern in Carto? Perhaps people are fine with hedges as linear features, there are some mappings? Perhaps because people are plain lazy? Perhaps, because we do not have a background map from which to copy colours? Who knows?

All the while, I do not see shrubbery mapping taking off here (Austria). Certainly not because the definition is too tight, I find it actually quite broad.

**Comment by jdhoek** [comment]
> Perhaps people are fine with hedges as linear features, there are some mappings? Perhaps because people are plain lazy? Perhaps, because we do not have a background map from which to copy colours?

In the Netherlands we do have access to such background maps under permissive licences, in combination with 8cm satellite imagery. (We are fortunate frontrunners in that area.) Combine that with the habit of using bushes and hedges as part of the design of streets and roads safe for bicycles, and you have an area where mapping shrubbery as areas makes a lot of sense (after all, the grassy areas and parking bays which together with the actual road and the shrubberies makes up most of the public street areas are mapped as areas as well). Linear hedges here are used mostly on private ground where such data is lacking or simply not interesting/relevant.

> Carto's choice is now to not render this tag at al; making for some quite odd maps when used and a mixed bag of other tags when people "map with the renderer in mind". In so far you want to create a seperate graphic representation for Shurbbery and what that would look like, I'll leave up to designers; but to not have it rendered at all seems odd.

It's silly. Even just rendering it the same as `natural=scrub` would improve the user experience without any downsides.

**Comment by OttoROSM** [comment]
@hungerburg for the example you give; it would actually depends on the kind usage on the ground. I could easily see this as shrubbery if indeed this is 'kept tidy' by humans; but it could also easily be scrub or part of re-forestation, and then I would tag it as such. (scrub/forest)

However if you look at Liebenstein, accross the street there is a patch accross the street of "Mühlbachler Installations- und Gebäudetechnik" that could be part of the garden of the house that is there, but could also be just a collection of shrubs planted by the village that would be mapped as Shrubbery. In that sense Shrubbery is to Shurbs as Forest is to Trees.

But have a look at 13 Böhmerwald Str. Freistadt, in front of the McDonalds; that would qualify a Shrubbery. Or across the street is a bit in between municipal garden or shrubbery. Then a little bit north in front of the bank (Promenade 11, 4240 Freistadt, Austria) is typical what would be Shrubbery.

**Comment by imagico** [comment]
> There are six shrubberies in all of western Austria, five of them by me. There is a suspicious cluster in Upper Austria - looking there, it is used to map reforestation, eg. https://www.openstreetmap.org/way/1169666435 - the note says: "Lt. OSM: Eine Fläche mit Sträuchern, die von Menschen aktiv gepflegt wird. = aufgeforstete Fläche" - in English - "Acc. to OSM: An area with shrubs that gets actively maintained by humans. = reforestation"

This matches my observation for most of the world outside the Netherlands (reminder: Netherlands accounts for more than half of the uses of the tag at the moment).  There are local concentrations mostly based on the work of individual mappers mapping either something locally specific (like the reforeststion areas you mention) or any and all urban or near urban greenery with that tag, probably often through armchair mapping where distinction between the other more commonly used tags is difficult.

Good demonstrator of such practice in a semi-urban context is the area around Backnang in Germany where a single mapper broadly mapped various streetside greenery (ranging from regularly mown grass via dwarf shrubs and larger bushes to full grown trees) with `natural=shrubbery`.  Like here:

https://mc.bbbike.org/mc/?lon=9.447153&lat=48.932972&zoom=17&num=2&mt0=bing-satellite&mt1=mapnik

This is not representative for use of the tag overall, most uses are in a more densely builtup and intensely maintained mid-urban or wealthy area context.  But is quite typical for the semi-urban/near-urban context.  This is a perfectly legitimate mapping paradigm in principle of course but it clashes both with consensus mapping practice in OSM in general and with the paradigm we use in OSM-Carto for differentiated rendering of vegetation (as discussed above in https://github.com/gravitystorm/openstreetmap-carto/issues/4473#issuecomment-1562652468)

There are other applications in use of the tag not yet mentioned, in particular:

* polygon mapping of individual shrubs (otherwise predominantly mapped with nodes tagged `natural=shrub`)
* instead of `man_made=planter`, sometimes in combination with `barrier=wall`/`barrier=retaining_wall`

**Comment by OttoROSM** [comment]
But is that a reason to not render it at all??

**Comment by OttoROSM** [comment]
As an example in the Netherlands, [this ](https://goo.gl/maps/rSut835uUaETY7Sw8) is an area between blocks of building. It is not a park, nor a garden, it is however maintained by the municipality. These are not scrubs, they are a collection of shrubs; with a two trees in it. 

On the other side of the street , you can see the fence of the residential gardens and some municipal green (i.e. shrubbery)
When you get into the [sidestreet](https://goo.gl/maps/hkk4tACyCxWPyeTm8), there is some parking spaces where the back is a municipal hedge and in between there is shrubbery. On the other side there is a municpal garden.

And this is common for the layout of villages and cities throughout the Netherlands.
This is area few years back was mapped where this all was tagged as garden. Which is not correct; it is not a garden.
Some people would tag it as Village Green, but apparently that is a very specific UK thing to tagged and is rendered. 

So can we at least get this rendered as something green; in stead of not getting it rendered at all.
I am sorry if in the rest of the world the imagery, Mappillary footage, additional layers or public/urban infrastructure is worse than in the Netherlands, but that should hardly be a reason to not show something :-) 

(and I travel quite a lot and know that urban infrastructure is easily as beautiful in Germany and other countries. Maybe not the imagery and layers to support the mapping accuracy.)

**Comment by imagico** [comment]
We render tags if and when (a) there is consensus among the maintainers that doing so is in support of our [goals](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CARTOGRAPHY.md) and (b) there is a PR suggesting a rendering for it that is suitable under these goals.  Neither of these is the case here.  Arguments for that assessment are provided in previous discussion (here and in #4530).  As mentioned already in https://github.com/gravitystorm/openstreetmap-carto/issues/4473#issuecomment-1562652468 we will be watching the development of use of the tag and we welcome further analysis of tag use as input (like @hungerburg did in https://github.com/gravitystorm/openstreetmap-carto/issues/4473#issuecomment-1627863712).

Slightly off-topic advice:  If the Netherlands mapping community considers themselves avant-garde w.r.t. mapping of vegetation or mapping in general it might be a good idea to develop your own map style reflecting that.  Many local communities have their own map style projects and diversity in available map styles is in general highly beneficial for OSM.  Our aim is to serve _the potential global map user_ and to support consensus among mappers globally - and that naturally limits our ability to adjust to local trends that are - subjectively or objectively - ahead of the rest of the world.

**Comment by OttoROSM** [comment]
Thanks for the advice; but as your goals state: Carto is "a major part of the public face of OpenStreetMap for many people the map on osm.org rendered with this style is OpenStreetMap." 

The goal here is not to invent yet another style, but to show on osm.org what is there in real life and being mapped.

Again, wether shrubbery requires it's own specific rendering as compared to scrub or not I leave to debate. But I do believe your goals support at least it being rendered at least similar to scrub or a shrub. And for the case where larger areas of i.e. hedges the barier=hedge tag really is not 'esthetically pleasing'

Showing where these green patches are
- **Legibility and Clarity** makes the map more 'intuitively readable' and 'esthetically pleasing'
- **Being understandable and supportive** 'serve as feedback for mappers and ecourage correct mapping' - the latter you could debate to what these patches really are; but certainly they are not scrubs, nor village green and garden is also not really what they are. But I guess this is something you could call 'of opinion' and this discussion should be part of the OSM discussion, rather than on the renderer.
- **Diversity** 'should represent the diversity of the OSM community and geography in general' we are not aiming to implement a specific tag rendering for something that only exists in the Netherlands. This type of green can be found all over the globe in urban environments. (and the tag is used also in other parts of the world). With that in mind , you could argue that the rendering of Village Green is kind of against this goal; as it is a very specific English tag (derived from English law), that has very limited match to what it means elsewhere and is often (mis)used in other countries to siginfy festival grounds or the main square in a village; and does not even have to include actual green.
- **A rich map**  'deliberately creating a fairly rich map showing a significant number of different features' and 'gives a broad recognition to the mappers' work' as well as 'not to show all or even most of the OSM data.' Shrubbery is a differentiation of green in urban environments, to differentiate between higher quality gardens and simpler shurbbery. But it does signify quite a large amount of area's on the map. 

The aspects of **Maintainability** and **Adaptability/EoU** I can't comment on. I can imagine that different rendering for Shrubbery density would be not support the goals; but to render it for now the same as scrub (in the same fashion grass and village green have the same rendering) or one specific rendering for shrubbery; I can't imagine would go against these goals.

**Comment by hungerburg** [comment]
@OttoROSM From your reply to my post I cannot make out if you are fine with the use of the scrubbery tag for reforestation or not. In case its fine, I see no chance for scrubbery as that makes it kind of a kitchen sink, in case it is not fine, I'd recommend to reach out to the mapper(s) and try to clarify what went wrong.

**Comment by pelderson** [comment]
Not scrubbery, but shrubbery, i.e. an area of shrubs. Shrubs are not trees.
Shrubbery can surround a tree, in that case map the tree in the shrubbery
area. If there are many trees with shrubbery in between, it will probably
get mapped as a landuse=forest or as natural=wood.
Peter Elderson


Op ma 10 jul 2023 om 23:46 schreef Hungerburg ***@***.***>:

**Comment by OttoROSM** [comment]
@imagico I saw your comment on heigth; personally I don't think this is necesarily something you would want to show in the layer. At least not on Carto. Differentiation in density could be usefull; especially to diferentiate between a 'box' style hedge that occupies a larger area vs a hedge used as a barrier and shrubbery used as urban greenery.

Again, the exact implementation of this tag in terms of visualisation; i am no expert in. I just find it odd that today it is not being rendered at all, where per your goals and gudielines; I do find merit in the differentiation between scrub (unmaintained wild undergrowth not being trees), shrubbery (properly maintained greenery not being gardens) and village greens (oddly English named centers of villages ;-), often having nothing to do with actual plants (asside from maybe a patch of grass))

**Comment by vincentvd1** [comment]
> We render tags if and when (a) there is consensus among the maintainers that doing so is in support of our [goals](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CARTOGRAPHY.md)

Rendering shrubbery would comply with at least goal 2 (**Being understandable and supportive for mappers**) and 4 (A rich map). People now incorrectly use `natural=scrub` for areas that are clearly `natural=shrubbery`. They do this solely because scrub renders and shrubbery not. And shrubbery is a common feature, especially in urban environments so rendering shrubbery would show the richness of the data in some cities.

**Comment by vincentvd1** [comment]
I did a small analysis of the current shrubbery use per country and continent

Some comments on this:
- There are a total of 24440 polygons with shrubbery
- In the Netherlands, we have detailed datasets and because of previous imports, we can focus a lot on micromapping like shrubbery. 
- It is also known that western Europe is more detailed so it also makes sense that in that region, there is higher usage. 
- The usage in the USA is also not small given an unrendered tag. It accounts for 15% of all shrubbery.
- Again, for an unrendered tag, the overal usage is high and steadily growing. 

## Per country

<!DOCTYPE html>

<html>


<body>


Country | Count | Percentage
-- | -- | --
Netherlands | 13016 | 53.257
United States of America | 3622 | 14.82
Germany | 1387 | 5.675
Belgium | 1099 | 4.497
Poland | 872 | 3.568
U.K. of Great Britain and Northern Ireland | 735 | 3.007
France | 690 | 2.823
Sweden | 470 | 1.923
Russian Federation | 329 | 1.346
China | 323 | 1.322
  | 229 | 0.937
Indonesia | 197 | 0.806
Denmark | 172 | 0.704
Canada | 123 | 0.503
Italy | 121 | 0.495
Austria | 118 | 0.483
Iran (Islamic Republic of) | 92 | 0.376
Switzerland | 90 | 0.368
Australia | 88 | 0.36
Finland | 67 | 0.274
Philippines | 63 | 0.258
Hungary | 59 | 0.241
Hong Kong | 48 | 0.196
New Zealand | 41 | 0.168
Spain | 40 | 0.164
Mexico | 33 | 0.135
Latvia | 31 | 0.127
Luxembourg | 30 | 0.123
Czech Republic | 29 | 0.119
Ukraine | 21 | 0.086
Slovakia | 19 | 0.078
Norway | 19 | 0.078
Democratic People's Republic of Korea | 17 | 0.07
Republic of Korea | 16 | 0.065
Djibouti | 15 | 0.061
Japan | 13 | 0.053
South Africa | 10 | 0.041
Slovenia | 10 | 0.041
Portugal | 10 | 0.041
Israel | 10 | 0.041


</body>

</html>



## Per continent
<!DOCTYPE html>

<html>


<body>


continent | count | Percentage
-- | -- | --
Europe | 19447 | 79.57
Africa | 52 | 0.213
Oceania | 130 | 0.532
Americas | 3781 | 15.471
Asia | 801 | 3.277
  | 229 | 0.937


</body>

</html>

**Comment by hungerburg** [comment]
> use per country

With three (3) taggings I am above median - https://overpass-turbo.eu/s/1yxG - in Austria.

**Comment by vincentvd1** [comment]
Ow, thanks for that overpass code. Didn't know that was possible. So, a further analysis then.
- In total 662 Used shrubbery 1 or more times.
- The average use per person is 36 with 191 people who used it more then 10 times. 

Analysis on al usage:
- The image below clearly shows more usage in Europe and Northen America. This is ofcourse to be expected. 
- If you zoom in on the areas in Europe and the USA, you can spot the cities and villages based on the cluster of points. This means that shrubbery is predominately used in urban areas. This is quite an accomplishment for an unrendered tag. 
- I see that in Africa, shrubbery is used where it clearly should be scrub. I will try to correct the clearly wrong use there. I cannot see when and by who this is mapped so I cannot give an explanation for that (e.g. is it mapped during the proposal process).
![image](https://github.com/gravitystorm/openstreetmap-carto/assets/10993040/3f59735d-bff2-4447-9816-6fb41d4c13ef)

**Comment by OttoROSM** [comment]
> This tag is currently only used in the Netherlands, England and a couple other spots. It needs to be much more widely used before it could be rendered here:
> 
> <img alt="Screen Shot 2021-09-18 at 23 08 05" width="729" src="https://user-images.githubusercontent.com/42757252/133917349-ddb04aaa-cc57-4478-bbf5-13bcbdfa7b7a.png">
> 
> I believe we should close this issue for now, but it could be reopened in a year or two if the situation has changed and the tag has been widel adopted by mappers in many countries.

Seems well time to reopen and implement.

**Comment by vincentvd1** [comment]
Is there an objection to render it the same as `natural=scrub`? It has been used 33.000+ times now and even renderers like Organic maps, openmaptiles and Tracetrack Topo render it. Rendering it the same as `natural=scrub` seems like a good solution for now. In the future, the rendering can be adjusted if needed.

**Comment by imagico** [comment]
Standing decision is not to render this.  Any new arguments on that matter are welcome of course.

On the hypothetical question of how we could proceed on rendering this in the current de facto meaning if we should think it is advisable to do so i commented a bit in https://github.com/gravitystorm/openstreetmap-carto/issues/4473#issuecomment-1562652468.  But as said there - this is purely hypothetical - tags do not tend to widen in use substantially without a change in meaning.

Current use numbers:  Global: 34030, Netherlands: 19619 (that means increase in the dominance of the NL from 53% reported in https://github.com/gravitystorm/openstreetmap-carto/issues/4473#issuecomment-1666555138 to 58%)  Also over-proportionally increasing seems to be the use in Flanders and Brussels (whole of Belgium 1646 uses - 4.8% from 4.5%).

As a frame of reference: During the same period of four months the use of other tags overlapping in de facto meaning with `natural=shrubbery` have increased by:

* `natural=scrub`: 150k
* `barrier=hedge`: 120k
* `leisure=garden`: 50k
* `natural=heath`: 17k
* `natural=shrub`: 12k

**Comment by pelderson** [comment]
> how we could proceed on rendering this 
A green colour like natural=wood or leisure=garden would do nicely for these mostly small areas of maintained shrubs. The tag is quite popular, despite not being rendered on OSM Carto.

**Comment by maro-21** [comment]
I support the idea!

The tag is already used 40,000 times and its usage is growing exponentially.
https://taghistory.raifer.tech/#***/natural/shrubbery

**Comment by hungerburg** [comment]
40k is not bad for a tag-as-you-like tag. The hockey stick though is mostly a trick of the zoom level you are looking at the data, try changing window size.

Curiously, within Europe, it is mostly used in North Macedonia? https://dashboard.ohsome.org/#backend=ohsomeApi&groupBy=boundary&time=2021-03-01T00%3A00%3A00Z%2F2024-03-19T14%3A00Z%2FP1M&key=natural&value=shrubbery&types=way&measure=area&adminids=-2323309%2C-52411%2C-2202162%2C-51477%2C-51701%2C-365331%2C-16239%2C-51684%2C-62149%2C-49715%2C-62273%2C-1311341%2C-295480%2C-50046%2C-52822%2C-2978650%2C-54224%2C-79510%2C-72594%2C-72596%2C-59065%2C-60199%2C-14296%2C-21335%2C-58974%2C-90689%2C-214885%2C-218657%2C-1741311%2C-186382%2C-192307%2C-53293%2C-53292%2C-2528142%2C-53296%2C-2088990

![Screenshot 2024-03-23 at 23-54-43 ohsome - dashboard](https://github.com/gravitystorm/openstreetmap-carto/assets/586972/bcc7d640-fa08-49d4-a15e-e3ee95ca9998)

**Comment by maro-21** [comment]
Not surprisingly, a tag is used more often in Western Europe - there are the most active users there.

**Comment by OttoROSM** [comment]
> **50k** is not bad for a tag-as-you-like tag.

![image](https://github.com/user-attachments/assets/a72edfed-1bf0-4042-b6d0-bceb4394abdf)

**Comment by OttoROSM** [comment]
Issue is still there. tag is not being rendered; while it is being used more and more.
The criterium from yourself was that you don't want to render tags that are not being used; it is being used.

**Comment by ZeLonewolf** [comment]
While the overall usage of this tag has continued to increase, it is still for the most part only used in significant numbers in The Netherlands and hasn't really caught on more broadly:

![image](https://github.com/user-attachments/assets/631703d5-5f72-4c05-af19-b05637e224a7)

You can use this [QLever query](https://qlever.cs.uni-freiburg.de/osm-planet/ZfN0Wz) to analyze this. Press "execute" and then "map view" to generate the map.

**Comment by OttoROSM** [comment]
Makes perfect sense; as for OSM in The Netherlands:

1. There is a lot of high quality up-to-date imagery (i.e. every 6 months, the country is overlflown by plane and provide 8cm imagery as open data) **and**
2. The Netherlands is densly populated and as a result highly regulated in terms of landuse and also this data is provided as open data by the goverement **and** (i.e. topographic data for landuse, buildings, water, etc...)
3. There is an active mapping comunity **and**
4. The Dutch Governement (and Dutch people) like the public space to be organised and 'taken care off'. So there is a lot of managed public green as planted/maintained shrubbery that are not parks or gardens. And it is hard to find an unmanaged things like scrub or forest. Most forest are managed as well.

Even though some countries have either one of the above; not many have the combination thereof. With the result that there is still a focus on getting buildings, roads and general landuse mapped; rather than detailed mapping where you'd make a distinction between landuse to this detail.

Thake this neighbourhood as an example; there is public grass and shrubbery next to the residential gardens.
![image](https://github.com/user-attachments/assets/2f923f7e-9ca4-406e-a60f-5e671b63dcf6)

**Comment by hungerburg** [comment]
I recently told a local mapper about shrubbery, because they mapped a 0.75m wide hedgerow as an area. Funnily, in OSM Carto this renders a 1.25m wide hedgerow. One week passed, no new shrubbery here. Even the 1.5 m wide hedge-areas still hedgerows with an eye inside. Wait a little longer?

**Comment by imagico** [comment]
Our limited ability to accommodate the Dutch avant-garde in mapping has already been explained in https://github.com/gravitystorm/openstreetmap-carto/issues/4473#issuecomment-1629165113.

Independent of that it would probably help anyone who wants to use `natural=shrubbery` in map rendering somewhere if the Dutch community would be able to provide pointers to some non-OSM Dutch maps that make a differentiation like the one made by the Dutch community through use of `natural=shrubbery` instead of more established tagging.  Meaning: Maps that show woody greenery with some level of maintenance independent of function and height in a distinct but uniform styling while showing woody greenery with a lower level of maintenance differently.

Side note: It always seemed curious to me why the Dutch community explicitly wants to map woody urban greenery undifferentiated by height considering height is the primary differentiation in traditional mapping of woody vegetation with no dedicated function in OSM (the natural=heath/scrub/wood progression) and has such immense practical significance.  But it makes sense considering the described mode of mapping: Even with very high resolution imagery it is hard to reliably determine the height of greenery when doing armchair mapping.  Especially for planted and intensely maintained stuff where you cannot hope to infer the height from the ecological setting.

@hungerburg - At the equator hedges at z19 are 1.2m wide, at 60 degrees latitude only half of that.  Ground unit rendering of line features at high zoom levels is something we have discussed in the past - but, unfortunately, only about 0.2 percent of all `barrier=hedge` features have a `width` tag at the moment.

**Comment by OttoROSM** [comment]
@imagico thank you for your response. Determining the height of any item in OSM is challenging and, in my opinion, still in its early stages. However, it is becoming more popular to accommodate 3D maps. As I mentioned in https://github.com/gravitystorm/openstreetmap-carto/issues/4473#issuecomment-1633067203; I don't believe height is the most crucial factor; perhaps density is more important, as it indicates whether you can walk through an area or not.

Regarding barrier=hedge, this tag is only for line items and doesn't accurately represent the irregular shapes of hedges. For this reason, the shrubbery area tag is a suitable alternative.

I wouldn't describe Dutch mapping as 'avant-garde'; rather, the Dutch government provides many excellent tools for detailed mapping that, to my knowledge, many other countries lack. Consequently, it is not surprising that these detailed differentiations are more commonly used. For example, aside from aerial imagery (including height profiles), most larger municipalities also publish street view images as open data. This is a great help in combining local surveys with armchair mapping.

I don't quite understand your curiosity about why one would want to differentiate between different types of vegetation. Carto also differentiates between various types of woodland/vegetation and land use, such as:
- At least seven different types of woody vegetation: natural, mangrove, broadleaved, needleleaved, mixed, leafless vegetation, and dwarf scrubs.
- Carto differentiates between farmland, vineyards, orchards and plant nurseries and even renders flowerbeds, which I always find to be a curious tag.
- Four types of sandy surfaces, including two types of beaches with distinctions between coarse sand (only 3.8k uses of this tag combination) and normal sand (39k uses of this combination).

Although I could at least think of one reason* of why you want to distinguish between these types of beaches, the tag usage for beach types is far less than for shrubbery. Yet, as Carto claims that _usage_ is a key motivator for rendering, there would be no reason to render coarse beaches. (* do I want to lay done on it or not ;-))

As @jdhoek comments here: https://github.com/gravitystorm/openstreetmap-carto/issues/4473#issuecomment-1628367042
> Even just rendering it the same as natural=scrub would improve the user experience without any downsides.

One could also think of adding a symbol for shrubs to the rendering of scrub; similar as you do for dog parks and playgrounds to distinguish them from other recreation grounds and sport centers. 

Why would it be strange to want to differentiate in tagging between shrubbery, garden, heath, and scrub, while Carto finds it perfectly normal to render with the same colour for differentiate tagging between grass, grassland, meadow, village green, golf course tee, fairway, and driving range?

As I mentioned, this is not a uniquely Dutch phenomenon. If you drive around other countries, you will see similar usage. However, before differentiating between vegetation, you first want to focus on other aspects. (e.g. roads and buildings)

My (our) request is to render the tag so it is visible as a green area on the map. If it can be distinguished from other vegetation, great; if not right now, perhaps in the future, that would be perfectly fine. But now we are left with a rendering that we grown to love (i.e. Carto) where parts of greenery in the urban area are absent from the map.

**Comment by maro-21** [comment]
> While the overall usage of this tag has continued to increase, it is still for the most part only used in significant numbers in The Netherlands and hasn't really caught on more broadly

Because this tag DOES depend on latitude. Not so much a tag as shrubbery. They occur in urban areas and I wouldn't expect them in Africa, for example. And it is not used in other places because there is no preset. And if there is no preset, mappers don't know about it. Preset or no preset affects usage. Advanced and experienced mappers who can dig up such a tag from the Wiki are in the minority. Most are occasional mappers who have 50 edits and will occasionally add something. They would add natural=shrubbery, but they don't know about it.

**Comment by hungerburg** [comment]
The tool mentioned by @ZeLonewolf finds some interesting mappings in Russia, e.g. https://www.openstreetmap.org/way/1227160205 - where do you think they got the tag from?

**Comment by maro-21** [comment]
> where do you think they got the tag from

From JOSM preset.

**Comment by Koreller** [comment]
This tag is very useful and deserves to be rendered. I've mapped areas with this tag, and because it isn't rendered, the map is much less attractive and less understandable.

It would be very great to render it !

**Comment by gdabski** [comment]
> Carto differentiates between farmland, vineyards, orchards and plant nurseries and even renders flowerbeds, which I always find to be a curious tag.

Ditto. Since `flowerbed` started to be rendered in Carto, I'm seeing people using that for what `shrubbery` should be, i.e. areas of non-grassy managed greenery with no flowers whatsoever.

And one could doubt if `flowerbed` even has or had a consistently understood or applied definition. At the time rendering was added, the Wiki [article for `flowerbed`](https://wiki.openstreetmap.org/w/index.php?title=Tag:landuse%3Dflowerbed&oldid=2506679)  actually displayed a picture of something that is not a flowerbed per the Wiki's definition.

**Comment by K4LCIFER** [comment]
Perhaps rendering [`landuse=shrubs`](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dshrubs) would be a better alternative than `natural=shrubbery`? There's already precedence with things like [`landuse=flowerbed`](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dflowerbed). 

I'm not sure putting it under a `natural` key is appropriate. I could certainly be wrong in my interpretation, but it seems that `natural=shrubbery` is used to describe an actively manicured display of shrubs This, to me, doesn't feel like something that fits under the `natural` key, as, iiuc, the `natural` key is meant for features that evolve "naturally" over time — something that is actively manicured dosen't feel like it evolves "naturally". Naturally evolving shrubbery seems to be adequately covered by `natural=scrub`.

`landuse=shrubs` is growing rapidly:

<img width="338" height="294" alt="Image" src="https://github.com/user-attachments/assets/814e74f5-8567-439d-959f-eade5dffa63d" />

and it seems to have a broad geographic distribution

<img width="740" height="383" alt="Image" src="https://github.com/user-attachments/assets/b6bfc007-e0d6-4604-b773-5d85c883400f" />

**Comment by realeatham** [comment]
Why is this closed as completed rather than as not planned

**Comment by IainTurpie** [comment]
Hi, I'm relatively new to contributing to OSM, and I wanted to add areas of shrubbery to the map. I can see that it isn't rendered right now, but I see no reason it shouldn't be? It seems to be widely used, more so than other features which are already rendered.
Why not reuse the pattern from landuse=flowerbed but with a darker colour for the pattern? I think the symbol would give enough distinction between other natural areas but a clear enough idea that it's a managed feature that it could work?

**Comment by mistenman** [comment]
Not rendering `natural=shrubbery` is a strange hill to die on. It's almost at 100k usage, and clearly distinct from the alternatives suggested. Scrub is uncultivated, shrubbery isn't. [Look at the plants under the pin here, for example.](https://maps.app.goo.gl/JKRyVx1DVBWAbxR47) What else would you call that? A flowerbed? The wider implementation of the tag is being held back by the refusal to render it. If you want to see it become de facto overnight, just render it, please. We'll have more accurate maps in both data and visuals.

**Comment by mistenman** [comment]
Here are `natural=` tags that are rendered by Carto despite having less usage and not being officially approved:

https://wiki.openstreetmap.org/wiki/Tag:natural=mud
https://wiki.openstreetmap.org/wiki/Tag:natural=shingle
https://wiki.openstreetmap.org/wiki/Tag:natural=bay

**Comment by LordGarySugar** [comment]
What is the reason for this issue still being marked as closed? natural=shrubbery is currently at 96,000 uses and growing all the time, and is one of the top most requested things I see that people wish were rendered on carto.

**Comment by imagico** [comment]
Please carefully read the previous discussion here before making new comments and only comment if you have meaningful new information to add that is of relevance for map design work here.  If you have a question please see if that has already been answered.  If you don't understand an explanation then ask specifically about it.

The issue discussions on this issue tracker are for style developers to coordinate their map design work.  This is not a lobbying platform.

**Comment by LordGarySugar** [comment]
The meaningful new information is that natural=shrubbery has close to 100,000 uses showing a clear acceptance of the tag amongst the community worldwide. This is a change from when the issue was (rightfully) closed in 2021 for having relatively few uses. The fact that the issue remains closed indicates the maintainers are not interested in adding support, so you're not going to get any suggestions for how it should be rendered if people are focussed on the fact the issue is closed instead. The last comment I can see from yourself is from 2024 where you described the use of natural=shrubbery as "the Dutch avant-garde in mapping". Since that time usages of this tag have doubled and spread widely across Europe and indeed the whole world. So I think that previous characterisation is now incorrect, and requires a re-appraisal of the original proposal. I also think it's pretty poor that the recent comments have been marked as off-topic. My question was directly related to this issue, asking why it can't be reopened due to the large amount of usage of this tag. It just comes across as silencing criticism in my opinion (not that my previous comment even was criticism. But this one is.)

**Comment by mistenman** [comment]
Preamble: it's understandable if you're unsure what kind of shrubbery `natural=shrubbery` is addressing. It might feel obvious to people grown up in areas where such features are common, but in case you need a visual primer, here's Google Images pages of such features from various localities when searched in [German](https://www.google.com/search?q=Abstandsgr%C3%BCn+Str%C3%A4ucher&udm=2), [Dutch](https://www.google.com/search?q=Vakbeplanting+heesters+openbaar+groen&udm=2), [Chinese](https://www.google.com/search?q=%E9%81%93%E8%B7%AF%E7%BB%BF%E5%8C%96+%E7%81%8C%E6%9C%A8+/+%E9%81%93%E8%B7%AF%E7%B6%A0%E5%8C%96+%E7%81%8C%E6%9C%A8&udm=2), and [Portuguese](https://www.google.com/search?q=Maci%C3%A7o+de+arbustos+urbano&udm=2). These search results do skew more towards very manicured examples like hedges or gardens, as they are far more photographable than non-interesting bushes, but I hope they contain enough diversity to understand what `natural=shrubbery` would be useful for that the other `natural=` tags aren't.

The discussion is fragmented and it is not clear what the prerequisites for reopening the issue are. My main request would be to have clear reasoning for why this issue remains closed and clear criteria for it to be re-opened. I'll quote and go through the comment I believe to establish the most clear reasoning for why the rendering is not implemented yet.

> A reminder to everyone: This issue tracker is for volunteer map designers to discuss map design work under the [goals of this style](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CARTOGRAPHY.md). The challenges of this w.r.t. this specific request have been outlined above so please limit your commentary to things that might help addressing these issues, specifically design ideas that might do so, or pointers to changes in the de-facto meaning of the tag that might be significant.

Rendering the feature would support all of the goals listed in [CARTOGRAPHY.md](https://github.com/openstreetmap-carto/openstreetmap-carto/blob/master/CARTOGRAPHY.md). The tag is at a de facto status, and has quickly growing implementation. 

> 
> I am going to try answering the question by [@pelderson](https://github.com/pelderson):
> 
> > How much growth do we need to get areas of shrubbery, mapped as natural=shrubbery, rendered?
> 
> Areas of shrubbery (according to how Wikipedia/OED define this term) are rendered when mapped in the currently predominant ways of mapping them in OSM, that is differentiated according to either height of vegetation or function (as explained in detail above):
> 
> `leisure=garden` - used for shrubbery (but also for other vegetation) planted and maintained systematically for decorative purposes.
> 
> `natural=heath` - used generically for low height shrubbery independent of function.
> 
> `natural=scrub` - used generically for tall shrubbery independent of function.
> 
> `barrier=hedge` - used for shrubbery planted in a linear arrangement to serve as a barrier of some sort.
> 
> `landuse=forest` - used sometimes (but not exclusively) for tall shrubs planted for wood production.
> 
> 
> `natural=shrubbery`, which is used as an alternative way to map shrubbery by some mappers, undifferentiated by either height or function and not exclusively for shrubbery but essentially for any kind of urban and near urban vegetation in height less than mature trees, is currently not rendered.

`natural=shrubbery` has its function clearly differentiated from all of these tags. The features that the tag fulfills are clearly distinguishable from gardens, heaths, hedges, and forests. The prevailing tag that fulfills the purpose of what `natural=shrubbery` should be used for is `natural=scrub`. `natural=scrub`, used to describe scrub, shrubland, scrubland, or brush, does not accurately describe the urban shrubbery it is often used for, and is being replaced by `shrubbery` in cases where `shrubbery` better describes the area. 

Scrub is not a generic term for tall shrubbery independent of function, even if current mapping practices are to use it for such features for lack of a more descriptive tag. Ideally, the word in the tag should reflect what it is representing. 
To add more credence to the argument that not all shrubbery is scrub, here are some major dictionary definitions of the word; dictionaries do not decree what a word means, but they generally reflect the consensus held by most speakers of the language:

Merriam Webster:
> vegetation consisting chiefly of [scrubs](https://www.merriam-webster.com/dictionary/scrubs)

Cambridge Dictionary:
> (an area of land covered with) short trees and bushes, growing on dry ground of low quality

Collins Dictionary:
> Scrub consists of low trees and bushes, especially in an area that has very little rain. 

Clearly, scrub has a distinct connotation.

Your comment continues:

> If the use of this tag would increase to a comparable or higher level than the other tags mentioned that we render in the domain of mapping areas of shrubbery without a change in de facto meaning (which is very unlikely to happen, typically tags change in meaning if their use widens) that would face us with a very difficult decision:
> 
> (1) we would either need to continue explicitly not rendering a tag with wide use because it cannot be consistently integrated into our current paradigm of rendering vegetation or
> 
> (2) we would need to completely abandon our current paradigm of rendering vegetation differentiated by physical characteristics and function and flatten all vegetation rendering in height less than mature trees to one uniform  design.  That would of course be a sad day for anyone interested in differentiated rendering of vegetation.

`natural=shrubbery` is at a de facto usage. It's not in the millions yet, but its adoption is growing rapidly. The claim about needing to abandon your paradigm of rendering vegetation differentiated by physical characteristics and function is not true, and adding `natural=shrubbery` actually strengthens your paradigm if people are to start using it instead of `natural=scrub` for planted, managed, low-growing ornamental woody vegetation that differentiates it from wild scrub (unmanaged/natural growth) and grass.

> Again - this is unlikely to happen because at least historically the world wide mapper community has often shown to be able to rectify gross inconsistencies in tagging paradigms like this once tags really get exposed to and scrutinized by the world wide community in all its diversity.
> 
> I am confident that this will happen also here. It could likely happen either by mappers re-defining and better differentiating and semantically delineating `natural=shrubbery` or by abandoning it in favor of other mapping ideas. I will continue to observe this and if and when some trend crystalizes here that we can positively support under our goals i would be more than happy to support anyone in developing a rendering solution for that in OSM-Carto.

`natural=shrubbery` is semantically delineated from the other tags you suggest, as has been demonstrated multiple times in previous comments, and as can be demonstrated more if necessary.

**Comment by mistenman** [comment]
Could a concrete answer regarding the matter be provided in order to advance the topic? @imagico

**Comment by imagico** [comment]
> Could a concrete answer regarding the matter be provided in order to advance the topic?

Answer to what question exactly?

The reason why this issue is closed is because we have a standing decision not to render this tag - see https://github.com/openstreetmap-carto/openstreetmap-carto/issues/4473#issuecomment-1837589849.  The issue being closed means that the maintainers do not consider it likely that a concrete change to add rendering will be accepted.  See also the two criteria mentioned in https://github.com/openstreetmap-carto/openstreetmap-carto/issues/4473#issuecomment-1629165113.  

As mentioned before this assessment can change if the practical use of the tag changes substantially in meaning, but it is not going to change based on lobbying or based on assertions without evidence.  I have looked at how tag use in this field develops several times over the last years and based on that my assessment remains that outside the Netherlands `natural=shrubbery` continues to not be the consensus tagging for anything specific.  Currently 55 percent of all `natural=shrubbery` are in the Netherlands - which is essentially unchanged as well (53 percent in August 2023, 58 percent in December 2023 - see https://github.com/openstreetmap-carto/openstreetmap-carto/issues/4473#issuecomment-1666555138 / https://github.com/openstreetmap-carto/openstreetmap-carto/issues/4473#issuecomment-1837589849).  If anyone has evidence for a well delineated consensus meaning in practical use in any local community outside the Netherlands presenting that here would certainly be valuable.

**Comment by vincentvd1** [comment]
> Currently 55 percent of all `natural=shrubbery` are in the Netherlands - which is essentially unchanged as well (53 percent in August 2023, 58 percent in December 2023 - see [#4473 (comment)](https://github.com/openstreetmap-carto/openstreetmap-carto/issues/4473#issuecomment-1666555138) / [#4473 (comment)](https://github.com/openstreetmap-carto/openstreetmap-carto/issues/4473#issuecomment-1837589849))

We in NL have access to highly detailed reference data so we will always be the #1 shrubbery user. The fact that this percentage didn´t change while the usage grew means that more people outside NL started using it. On the [tag info map](https://taginfo.openstreetmap.org/tags/natural=shrubbery#map) you see that western europe has a dense usage which matches the expectation that western europe in general is more densely mapped. At the [taginfo project page](https://taginfo.openstreetmap.org/tags/natural=shrubbery#projects) you also see that more and more projects have presets for it. Also more renderers like Organic maps and openstackTopo render it.

It will also always be a chicken and egg story. Carto is the prominent renderer where mappers will look at. This way there is also no perspective when the tag will be rendered. TagInfo clearly shows a steady grow in multiple countries (and yes, as explained mainly western europe).

 What if we render it very simply like with the wood color with a shrubbery pattern (like jeroen originaly proposed). No variation on shrubbery density or so what, just simple. Would such a PR work?

**Comment by hungerburg** [comment]
In my local area, a town of 150.000 inhabitants, there are 35 shrubberies mapped, mostly closet sized urban greenery. All by the same mapper, a non-local that maps mostly area:highway from aerial and mapillary. He seems to be quite fond of the French-style, because they render area:highway. Curiously, that style does not render shrubberies. How about reaching out to them?

PS: The invitation "A slightly wilder look is also possible." really intended?

**Comment by IainTurpie** [comment]
<img width="500" height="500" alt="Image" src="https://github.com/user-attachments/assets/6b97ed13-f4c1-48dd-a960-105f3abf0d8a" /> <img width="500" height="500" alt="Image" src="https://github.com/user-attachments/assets/e579b73c-b20a-4965-8be6-3345c3ac1dd7" /><img width="500" height="500" alt="Image" src="https://github.com/user-attachments/assets/4951d3ed-921f-4c70-bece-a5d349b06250" />

A small suggestion of how I imagine shrubbery could look? Couple options for colours provided. Shrubbery is the darker patches (such as the larger one near the top). This is from a village I've been mapping in the Scottish Borders called [Cardrona](https://www.openstreetmap.org/#map=16/55.63710/-3.10514).
